internal/expr/parser.go

```diff
@@ -779,11 +781,11 @@ func (p *Parser) parseMemberInputExpr() (*memberInputExpr, bool, error) {
 func (p *Parser) parseAsteriskInputExpr() (*asteriskInputExpr, bool, error) {
 	cp := p.save()
 
+	parenCol := p.colNum()
+	parenLine := p.lineNum
```

Comment:
I think this is redundant now and you could do:
```
cp.lineNum
cp.colNum()
```
without having to store it explicitly.

Reply:
Good point